src/helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { Person } from './types';
+
+export const getDetailsOfParents = (peoples: Person[]): Person[] => {
+  return peoples.map(person => {
```

Comment:
There is a typo in the prop `peoples`. It should be `people` as it is more grammatically correct.